app/background/api/cosmeticFilterAPI.ts

```diff
@@ -0,0 +1,41 @@
+// import { Tab } from '../../types/state/shieldsPannelState'
+// import { getShieldSettingsForTabData, getTabData } from './shieldsAPI'
```

Comment:
let's remove these imports as we don't need them

Reply:
done